### DISCUSSION TITLE
Difficulties Installing Imagemagick 7

### DISCUSSION THREAD

**Discussion by lm-sudo** [ImageMagick/ImageMagick#6865] [Help]
### ImageMagick version

6.9.11-60 Q16

### Operating system, version and so on

Ubuntu 20.04.6 LTS

### Description

Hello, I've recently been working with imagemagick 7 on WSL, I was able to install it by following this guide: https://doc.owncloud.com/server/next/admin_manual/installation/manual_installation/manual_imagick7.html
I followed the instructions exactly under the header titled: Install the Latest ImageMagick-7 Binary.

I am now working on a fresh WSL instance and am trying to install imagemagick 7 on it but am getting unexpected behavior. I follow the same steps but now several parts of the installation process seem to be getting skipped by the image magick easy installer. I have attached a picture that shows this. 

Any ideas on what may be causing this? And if not, should I try a different approach to installation? Thank you.
![weird_behavior](https://github.com/ImageMagick/ImageMagick/assets/89110072/f0605204-0130-4211-b8d9-49d3b6b7e84d)

**Comment by fmw42** [comment]
You do not show any errors?  Look in the config.log file for any errors. Then fix them. Then try again.

**Reply by lm-sudo** [reply]
Evidently it appears if you try to run the IMEI tool through the "Ubuntu" windows app which auto launches your current WSL linux instance, this error occurs. I tried running the IMEI tool through windows powershell which can also allow you to access your WSL linux instance and I was successfully able to install image magick 7 through there. I wasn't able to find any info on this anywhere in this discussions forum so hopefully this will help someone at least.

**Reply by dlemstra** [reply]
We did not write and do not maintain the IMEI tool so you will need to ask your question elsewhere?
